core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceTest.java

```diff
@@ -672,6 +675,46 @@ public String headerWithoutValue(RequestContext ctx,
         }
     }
 
+    @ResponseConverter(UnformattedStringConverterFunction.class)
+    public static class MyAnnotatedService12 {
+
+        @Get
+        @Path("/multiGet1")
+        @Path("/multiGet2")
+        public String multipleGet(RequestContext ctx) {
+            return "multiGet";
+        }
+
+        @Get
+        @Path("/duplicateGet")
+        @Path("/duplicateGet")
+        public String duplicateGet(RequestContext ctx) {
+            return "duplicateGet";
+        }
+
+        @Get
+        @Path("/multiGetWithParam1/{param}")
+        @Path("/multiGetWithParam2/{param}")
+        public String multiGetWithParam(RequestContext ctx, @Param String param) {
+            return param;
+        }
+
+        @Get
+        @Path("/multiGetDiffParam1/{param1}")
+        @Path("/multiGetDiffParam2/{param2}")
+        public String multiGetDiffParam(RequestContext ctx, @Param String param1, @Param String param2) {
+            return "multiGetDiffParam";
```

Comment:
Could you return a string made from `param1` and `param2` and make sure the both path patterns are handled correctly?